### DISCUSSION TITLE
Customize launcher page to display a README

### DISCUSSION THREAD

**Discussion by kayhman** [jupyterlite/jupyterlite#1134] [Q&A] [answered]
Hello,

Thanks for your great job. I'm using Jupyter Lite to make the code associated with one of my books easily available online.

It works perfectly for me. However, I'd like to change the landing page. Currently, Jupyter Lite starts with the launcher opened.

I want to change that so that a README is open by default. Is that possible? 

I've been looking at the doc and tried multiple things, but I always land on the launcher.

Does somebody have an answer?

Thanks,

www.70mathematicalconcepts.com

**Comment by jtpio** [comment, marked as answer]
Thanks @kayhman for sharing your website :+1: 



> I want to change that so that a README is open by default. Is that possible?

It's possible to open a file automatically by specifying `?path=example.md` in the URL: https://jupyterlite.readthedocs.io/en/latest/howto/content/share.html

However the markdown file will open with the text editor by default and not the markdown preview.

An alternative could be to create a notebook and put some content in a markdown cell, and open that notebook by default.

**Reply by kayhman** [reply]
Thanks for your answer. By default, the page opened by default is `lab/index.html`. 

Do you know is there is a place in the config where I an change that to point to my readme?

I've also tried to add a redirect rule on Cloudflare, but it doesn't work either.

Thanks,
